domains/certificates/Query.API/API/MeasurementsSyncer/MeasurementsSyncService.cs

```diff
@@ -34,13 +37,20 @@ public MeasurementsSyncService(ILogger<MeasurementsSyncService> logger, ISliding
         _measurementSyncMetrics = measurementSyncMetrics;
         _measurementSyncPublisher = measurementSyncPublisher;
         _meteringPointsClient = meteringPointsClient;
+        _options = _measurementSyncOptions;
     }
 
     public async Task FetchAndPublishMeasurements(MeteringPointSyncInfo syncInfo, MeteringPointTimeSeriesSlidingWindow slidingWindow,
         CancellationToken stoppingToken)
     {
         var synchronizationPoint = UnixTimestamp.Now().RoundToLatestHour();
-        var fetchedMeasurements = await FetchMeasurements(slidingWindow, syncInfo.MeteringPointOwner, synchronizationPoint, stoppingToken);
+        var fetchToTimestamp = UnixTimestamp.Min(synchronizationPoint, synchronizationPoint.Add(TimeSpan.FromHours(-_options.Value.MinimumAgeThresholdHours)).RoundToLatestHour());
```

Comment:
Should this have the same guard (_if (_options.Value.MinimumAgeThresholdHours > 0)_) as below? Or maybe this should always be true so _if_ statement in line 48 does nothing?

Reply:
We fixed this together I believe. Should this be resolved?